app/templates/components/order-summary.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="ui segments">
+  <div class="ui secondary segment">
+    <h3 class="weight-400">{{t 'Order Summary'}}</h3>
+  </div>
+  <table class="ui very basic tablet stackable table" style="padding:10px">
+    <thead>
+      <tr>
+        <th class="four wide">{{t 'Ticket Type'}}</th>
+        <th class="four wide">{{t 'Price'}}</th>
+        <th class="ui aligned two wide">{{t 'Fee'}}</th>
+        <th class="one wide">{{t 'Quantity'}}</th>
+        <th class="ui aligned two wide">{{t 'Subtotal'}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <div class="ui small">
+            {{t 'Student/In'}}
```

Comment:
@mariobehling Since this PR deals only with the User Interface of the page. We're not currently fetching the data from the API. All this will be handled separately while integrating the page with the API.